lib/uffizzi/config_file.rb

```
@@ -86,7 +86,10 @@ def read
           acc.merge({ key.strip.to_sym => value.strip })
         end
       rescue Errno::ENOENT => e
-        Uffizzi.ui.say(e)
+        file_path = e.message.split(' ').last
+        message = "Configuration file not found: #{file_path}\n" \
+        'To configure the uffizzi CLI interactively, run $ uffizzi config'
+        Uffizzi.ui.say(message)
```

Looks like we need to raise an error